src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/customer/index_response.json

```diff
@@ -0,0 +1,144 @@
+                        "enabled": true
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@",
+                "_links": {
+                    "user": {
+                        "href": "@string@.contains('\/api\/users\/')"
+                    }
+                },
+                "_embedded": {
+                    "user": {
+                        "id": @integer@,
+                        "username": "@string@.isEmail()",
+                        "enabled": true
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@",
+                "_links": {
+                    "user": {
+                        "href": "@string@.contains('\/api\/users\/')"
+                    }
+                },
+                "_embedded": {
+                    "user": {
+                        "id": @integer@,
+                        "username": "@string@.isEmail()",
+                        "enabled": true
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@",
+                "_links": {
+                    "user": {
+                        "href": "@string@.contains('\/api\/users\/')"
+                    }
+                },
+                "_embedded": {
+                    "user": {
+                        "id": @integer@,
+                        "username": "@string@.isEmail()",
+                        "enabled": true
+                    }
+                }
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@"
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@"
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@"
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
+                "last_name": "@string@"
+            },
+            {
+                "id": @integer@,
+                "email_canonical": "@string@.isEmail()",
+                "first_name": "@string@",
```

Comment:
I'd prefer to use specific names/lastnames. That would be nicer than matching everything by pattern.